mithril-common/src/crypto_helper/types.rs

```diff
@@ -1,44 +1,89 @@
-use blake2::{digest::consts::U32, Blake2b};
-use ed25519_dalek;
-use mithril::key_reg::KeyReg;
+use crate::crypto_helper::cardano::{
```

Comment:
Parameter names are getting longer since we have a lot of resembling parameters. However, it sometimes gets confusing to track the code and decide which is which. I don't think we should change the names, but maybe the parameter documentation should give a little more detail.